core/api/jrpc/value_converter.hpp

```diff
@@ -50,7 +50,8 @@ namespace kagome::api {
   }
 
   inline jsonrpc::Value makeValue(const common::Buffer &v) {
-    return v.toVector();
+    static const std::string prefix("0x");
+    return prefix + v.toHex();
```

Comment:
To fix how we return buffers. They should be returned as 0x prefixed hex strings. As example getStorage rpc: https://github.com/w3f/PSPs/blob/psp-rpc-api/psp-002.md#state_getstorage